tests/Unit/Rules/ViolationsTest.php

```diff
@@ -72,4 +72,36 @@ public function test_to_string(): void
 
         $this->assertEquals($expected, $this->violationStore->toString());
     }
+
+    public function test_get_iterable(): void
```

Comment:
I suppose that the type checker already check this?

Reply:
Yes, I was trying to explicit it well. I would like to be more specific, any suggestions @fain182 ?

Reply:
@fain182 I changed the assertion with a more proper one, what do you think?

Reply:
much better :+1: